tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/SingleBalFileTestCase.java

```diff
@@ -94,6 +94,19 @@ public void testSingleBalFileWithJDBCUsage() throws BallerinaTestException {
         fooRunLeecher.waitForText(10000);
     }
 
+    @Test(description = "Test running bal with relative path")
+    public void testRunWithRelativePath() throws BallerinaTestException {
```

Comment:
Do we need an integration test as well ?

Reply:
I have written an integration test for run command and for build command, unit test is written.